### DISCUSSION TITLE
possible to combine more information?

### DISCUSSION THREAD

**Discussion by dnielso5** [allinurl/goaccess#2761] [General]
Maybe im missing something but it would be nice to click on a day and see what IP's accessed what site, and how much data.

for example in this SS:

![image](https://github.com/user-attachments/assets/472f4a0c-a74b-4e75-abe2-c8ed45af0cb9)

Id like to click on a date and see what IP's connected and what site they accessed

**Comment by allinurl** [comment]
#117 — [stay tuned](https://x.com/goaccess), working on it as we speak.
